src/elements/content-sidebar/ContentSidebar.js

```diff
@@ -66,6 +66,7 @@ type Props = {
     messages?: StringMap,
     metadataSidebarProps: MetadataSidebarProps,
     onAnnotationSelect?: Function,
+    onFetchFileSuccess?: Function,
```

Comment:
I think we should start using the arrow function notation for function types to be more explicit. Dont have to change the other props, just the new one if possible